Makefile

```diff
@@ -0,0 +1,13 @@
+clean:
+	rm -f *.rpm
+
+rpm:
+	mkdir -p packaging/root/usr/bin
+	cp *.py packaging/root/usr/bin
+	fpm -s dir -t rpm \
+            -n python-snappy \
+            --after-install=./packaging/after-install.sh \
+            --after-remove=./packaging/after-remove.sh \
+            --config-files=/etc/snappy.ini \
+            --rpm-use-file-permissions \
+            -C packaging/root . 
```

Comment:
Consider adding:
```
	    --rpm-user=root \
	    --rpm-group=root \
	    --license "MPL" \
	    --url "https://github.com/mozilla/Snappy-Symbolication-Server" \
	    --category "Applications/System" \
	    -a all \
	    --depends python \
	    --description "The Snappy Symbolication Server is a Web server for symbolicating Firefox stacks." \
```
The important args are `user` and `group` since without them the files will assume the ownership of whomever built them.